src/Tribe/Tickets.php

```diff
@@ -1389,5 +1393,15 @@ public static function get_login_url() {
 			 */
 			return apply_filters( 'tribe_tickets_ticket_login_url', $login_url );
 		}
+
+		/**
+		 * @param $return
```

Comment:
Hmm, ok. Maybe it's just me but `$return` doesn't seem like the best name in that case (it possibly makes more sense at the call site, ie in the `has_permission()` method) but right here it doesn't really reflect what it does.